Gopkg.toml

```diff
@@ -20,3 +20,6 @@
 #  name = "github.com/x/y"
 #  version = "2.4.0"
 
+[[override]]
```

Comment:
According to https://github.com/golang/dep/blob/master/docs/Gopkg.toml.md, `constraint` should be sufficient, shouldn't it?

Reply:
Okay. This is new territory for me.

Reply:
Indeed, tested with `constraint` and `dep ensure` puts the same stuff in `vendor/`